app/components/action-feed/template.hbs

```diff
@@ -1,18 +1,16 @@
-{{#if actionsList.isPending}}
-    <div class='loading'>
-        <div class="text-center p-v-md ball-scale ball-dark" aria-label="Loading data">
-            <div></div>
-        </div>
-    </div>
+{{#if listLoading}}
+    {{loading-indicator classes='ball-dark text-center loading'}}
 {{else}}
     {{#each actionsList key='id' as |action|}}
         {{action-feed-entry action=action toDetail=toDetail}}
+    {{else}}
+        <div class="text-center">{{t 'components.action-feed.no_actions'}}</div>
     {{/each}}
-    {{#if moreActions}}
+    {{#if (and moreActions (not loadingMore))}}
```

Comment:
This logic will display the loading indicator if `!moreActions`, which is wrong (go to the last page of actions, the loading indicator stays there). Seems like it should be something like:
```hbs
{{#if loadingMore}}
  {{loading-indicator}}
{{else if moreActions}}
  See more
{{/if}}
```